integration-tests/actions/vrfv2plus/vrfv2plus_config/config.go

```diff
@@ -0,0 +1,34 @@
+package vrfv2plus_config
+
+import "time"
+
+type VRFV2PlusConfig struct {
+	ChainlinkNodeFunding            float64 `envconfig:"CHAINLINK_NODE_FUNDING" default:".1"`                     // Amount of native currency to fund each chainlink node with
```

Comment:
IMO we don't need two configuration frameworks, please move all these vars to `config.toml`.
If you are varying different params, like `WrapperGasOverhead`, please create a separate group in `TOML` and name it, like `[SoakWords10]` or `[SoakWords100]`.
The goal here is to have 3 groups of vars:
- selectors, like `SELECTED_NETWORKS` to select some group in `TOML` and start a test
- secrets, we can't put them in `TOML` under git, you can use `envconfig` if you have a reason for that but the simplest way is just to use env vars in `config.go` and then put them in `PerformanceConfig` in runtime so you can have all vars in one struct you can easily dump locally
- `TOML` configuration reflecting all possible test configurations, if you have a particular case, for example varying amount of words, just put it there as a different test. It's much easier to share configuration in this way, I can open your tests and just try to run all the cases already configured.

Reply:
The goal here is to write as little documentation as possible but shit ready to run tests, describe only `TOML` config, and have a predefined set of tests that anyone can run without trying to guess the configuration. It'd be harder if we had it in two places, one `TOML` is enough for everything and only one selector for network, a test name, and all the secrets (that have no defaults) can be used as env vars.

Reply:
as agreed, will add new ticket to move to profile approach (toml config)